packages/landing/src/components/layer.switcher.dropdown.tsx

```diff
@@ -4,13 +4,13 @@ import { LayerInfo, MapConfig } from '../config.map.js';
 
 export interface LayerSwitcherDropdownState {
   layers?: Map<string, LayerInfo>;
-
+  extend: boolean;
```

Comment:
this should be `extent`? or possibly `zoomToExtent`

Reply:
Fixed.